phinx.json

```diff
@@ -0,0 +1,36 @@
+{
+  "paths": {
+    "migrations": "%%PHINX_CONFIG_DIR%%/migrations"
```

Comment:
Phinx currently [requires YAML](https://github.com/robmorgan/phinx/blob/cd2009b71e16ca8363e5e95ee19a991ae4cfcb5a/composer.json#L23). Regardless, you're right that anyone wanting to use YAML would have the extension installed.